### PR TITLE
[WIP] Delete command message from user

### LINKED ISSUE #276: Delete command message from user
Add an option to make the bot delete the message you sent like 
``s!play something`` and that message would be deleted. (If set to true in the guild config)

### PR DESCRIPTION
**Pull request by Doges**
Closes #276

### CODE REVIEW

**Comment by pedrofracassi** [comment]
What still needs to be done here?

**Comment by Doges** [comment]
look at the strings

File: src/modules/ConfigurationModule.js [hunk 1]
```diff
@@ -22,4 +22,11 @@ module.exports = class ConfigurationModule extends Module {
   async setLanguage (_guild, language) {
     await this._guilds.update(_guild, { language })
   }
+
+  async setDeleteUserMessage (_guild, state) {
+    const guild = await this._guilds.get(_guild, 'deleteUserMessage')
+    if (guild.deleteUserMessage === state) throw new Error('SAME_STATE')
+    guild.deleteUserMessage = state
+    await guild.save()
```
Comment (dpaiv0) [inline src/modules/ConfigurationModule.js]: replace l.29 and l.30 with `await this._guilds.update(_guild, { deleteUserMessage: state })`

Comment (Doges) [reply src/modules/ConfigurationModule.js]: yea, PORQUE TU NÃO REVISOU ESSA PORRA NO JOINLOCK, mas eu vou mudar, relaxa

**Comment by Doges** [comment]
remaking this with the new system l8

**Pull request closed without merging**